### ISSUE TITLE
Uncertain if specifying api-id is just unclear or broken when adding permissions to a role

### ISSUE THREAD

**Issue by atsvetkov** [auth0/auth0-cli#348]
### Description

When trying to add existing permissions to an existing role via CLI, I'm always getting back `404 Not Found: The resource server does not exist: ''` error. The documentation is a bit unclear regarding which ID of the API needs to be passed (the system-generated one or the user-friendly one), but the command fails in both cases anyway.

### Reproduction

```bash
$ auth0 roles permissions add rol_someroleid -a numeric_api_identifier -p "permission1,permission2"
OR
$ auth0 roles permissions add rol_someroleid -a user_friendly_api_identifier -p "permission1,permission2"

404 Not Found: The resource server does not exist: ''
```

### Environment
- auth0 version 0.10.2 711dbbda2fc5ed92d3e7c1467e301d615e19397f
- Windows 10, CLI running in WSL2

When running `auth0 roles permissions add` in the interactive mode, selecting API, role and permission goes fine and the command successfully associates the specified permissions to role. My main question is, how to correctly specify the API identifier and what else could be causing the error? Tried this on different tenants - same result.

Thank you!

**Comment by Widcket** [comment]
Hi @atsvetkov, thanks for raising this.
The command seems to work fine when done interactively:

<img width="749" alt="Screen Shot 2021-09-03 at 23 52 33" src="https://user-images.githubusercontent.com/5055789/132080238-3194bb87-84fd-4d02-9820-be53dd3d2f64.png">
<img width="759" alt="Screen Shot 2021-09-03 at 23 54 34" src="https://user-images.githubusercontent.com/5055789/132080241-b34cf4a8-b571-42db-9526-929b112784cf.png">

But falls apart when done with flags:

<img width="746" alt="Screen Shot 2021-09-03 at 23 56 24" src="https://user-images.githubusercontent.com/5055789/132080251-d35a4961-18be-443d-9b20-3dadda7d43f7.png">

I'll be taking a look a this.

**Comment by Widcket** [comment]
This is now fixed in [v0.11.0](https://github.com/auth0/auth0-cli/releases/tag/v0.11.0).
